sbe-tool/src/main/java/uk/co/real_logic/sbe/ir/IrDecoder.java

```diff
@@ -97,7 +99,7 @@ public Ir decode()
             i = captureHeader(tokens, 0);
         }
 
-        final Ir ir = new Ir(irPackageName, irNamespaceName, irId, irVersion, semanticVersion, irHeader);
+        final Ir ir = new Ir(irPackageName, irNamespaceName, irId, irVersion, semanticVersion, byteOrder, irHeader);
```

Comment:
What if the existing encoded IR was in BIG_ENDIAN format? Will this not result in a bug?